### PR TITLE
added sponsors to homepage

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: frontend/src/components/CollectiveCard.js [hunk 1]
```diff
@@ -16,41 +14,37 @@ export default class CollectiveCard extends Component {
   }
 
   render() {
-    const {id, bg, logo, name, mission, backerCount, sponsorCount, monthlyIncome, currency} = this.props;
-    const formattedMonthlyIncome = formatCurrency(monthlyIncome/100, currency, { compact: true, precision: 0 });
+    const {key, bg, logo, name, description, url, stats, className} = this.props;
+    if(stats.length === 2)
```
Comment (asood123) [inline frontend/src/components/CollectiveCard.js]: missing space

File: frontend/src/containers/HomePage.js [hunk 2]
```diff
@@ -13,9 +12,31 @@ import CollectiveCard from '../components/CollectiveCard';
 import fetchHome from '../actions/homepage/fetch';
 
 const mapCollectiveCardProps = group => {
-  group.sponsorCount = filterCollection(group.backers, {tier: 'sponsor'}).length;
-  group.backersCount = filterCollection(group.backers, {tier: 'backer'}).length;
-  group.monthlyIncome = (group.yearlyIncome / 12);
+  group.mission = `We are on a mission to ${group.mission}`;
+  group.stats = [];
+
+  if(group.contributors && Object.keys(group.contributors).length > 0)
```
Comment (asood123) [inline frontend/src/containers/HomePage.js]: missing space

File: frontend/src/containers/HomePage.js [hunk 3]
```diff
@@ -104,19 +126,37 @@ export class HomePage extends Component {
...
           </div>
           <div className='cta'>
             <div className='text'>We are slowly letting in new kinds of collectives</div>
-            <a href="#"><div className='button color-green'>join the waiting list!</div></a>
+            <div className='button color-green'><a href="mailto:info@opencollective.com?subject=join%20waiting%20list">join the waiting list!</a></div>
+          </div>
+        </section>
+        <section className='HomePageSponsors blue-gradient'>
+          <div className='heading'>Sponsors</div>
+          <div className='subheading'>Collectives do amazing things for their communities thanks to those awesome sponsors.</div>
```
Comment (asood123) [inline frontend/src/containers/HomePage.js]: s/those/these

File: frontend/src/containers/HomePage.js [hunk 4]
```diff
@@ -143,7 +183,7 @@ export class HomePage extends Component {
               </div>
             </div>
           </div>
-          <a href='#'>Learn more about the collectives</a>
+          {false && <a href='#'>Learn more about the collectives</a> }
```
Comment (asood123) [inline frontend/src/containers/HomePage.js]: why not just take this out altogether? It's not hard to add later.

File: frontend/src/containers/HomePage.js (hunk 2 above)
Comment (xdamman) [inline frontend/src/containers/HomePage.js]: can we make this into the eslint?

File: frontend/src/containers/HomePage.js (hunk 4 above)
Comment (xdamman) [inline frontend/src/containers/HomePage.js]: It was to still have it there and make sure we know how to add it back quickly. 
Is that bothering?

File: frontend/src/containers/HomePage.js (hunk 2 above)
Comment (asood123) [inline frontend/src/containers/HomePage.js]: it is already. https://github.com/OpenCollective/eslint-config-opencollective/blob/master/index.js. We need to upgrade it in package.json.

File: frontend/src/containers/HomePage.js (hunk 4 above)
Comment (asood123) [inline frontend/src/containers/HomePage.js]: Creates unnecessary code to parse through. I can understand if it's significant code or a test that's annoying to rewrite. These seemed like an easy line to delete and bring back when we need it.

**Pull request merged**